birdbuddy/client.py

```diff
@@ -40,21 +40,27 @@ class BirdBuddy:
     """Bird Buddy api client"""
 
     graphql: GraphqlClient
-    _email: str
-    _password: str
+    _email: str | None
+    _password: str | None
     _access_token: str | None
     _refresh_token: str | None
     _me: BirdBuddyUser | None
     _feeders: dict[str, Feeder]
     _collections: dict[str, Collection]
     _last_feed_date: datetime
 
-    def __init__(self, email: str, password: str) -> None:
+    def __init__(self, emailOrToken: str, password: str | None = None) -> None:
```

Comment:
```
def __init__(
    self,
    email: str | None = None,
    password: str | None = None,
    /,
    access_token: str | None = None,
    refresh_token: str | None = None,
) -> None:
    # then replace lines 56-57 to assign their values from the kwargs
```

Then we can use it like this:
```
client = BirdBuddy(refresh_token="your manual token")
await client.refresh()
```

Reply:
I made these changes and also merged in master.

Mind testing this change using your manual refresh token method, like shown above?

Reply:
Yep, it works.

Reply:
Great! I'll get this merged, thanks!